client/src/components/payment/payment-mixin.ts

```diff
@@ -17,6 +17,10 @@ export default class PaymentMixin extends Vue {
     return this.$store.getters[paymentTypes.GET_SBC_PAYMENT]
   }
 
+  get sbcPaymentStatus () {
+    return this.$store.getters[paymentTypes.GET_SBC_PAYMENT_STATUS]
```

Comment:
At some point, "paymentTypes" should be changed to PascalCase so it's not confused with a property (which should be camelCase).

Reply:
Not enforced by eslint, and those are not actually 'types' referenced in those files, they're exported string constants so technically they are just exported variables. I could have written 'action_dictionary.<type>' or 'all_actions.<type> I guess but yeah...

Reply:
You're right, but they're used as types/enum.

Please fix if you're in this area again.